app/controllers/import_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ImportController < ApplicationController
+  def new
+    @import = Import.new
+  end
+
+  def create
+    @import = Import.new(import_params)
+
+    if @import.save
+      redirect_to root_url
```

Comment:
Probably should render a success or failure message only so it's clear to the user if the upload is a success or not.

should not ever display or llst users in the UI - it should only possible via the API with an access token.